ios/REANodesManager.mm

```diff
@@ -211,19 +215,17 @@ - (instancetype)initWithModule:(REAModule *)reanimatedModule uiManager:(RCTUIMan
     _viewRegistry = [_uiManager valueForKey:@"_viewRegistry"];
     _shouldFlushUpdateBuffer = false;
   }
+#endif
+  [[self getDisplayLink] addToRunLoop:[NSRunLoop mainRunLoop] forMode:NSRunLoopCommonModes];
+  [[self getDisplayLink] setPaused:true];
```

Comment:
Can we move this to `getDisplayLink` and only call `[self getDisplayLink]` here?